src/parser/core/stats.test.jsx

```diff
@@ -64,12 +64,12 @@ describe('stats', () => {
   it('scales secondary stat correctly', () => {
     // Spoils of Neltharus
     // https://www.wowhead.com/item=193773/spoils-of-neltharus
-    const spoilsOfNeltharus = (itemLevel) => calculateSecondaryStatDefault(250, 547.57, itemLevel);
+    const spoilsOfNeltharus = (itemLevel) => calculateSecondaryStatDefault(250, 482, itemLevel);
 
-    expect(spoilsOfNeltharus(382)).toBeWithin(2253.943, 1); // Normal
-    expect(spoilsOfNeltharus(402)).toBeWithin(2528.523, 1); // Mythic+
-    expect(spoilsOfNeltharus(441)).toBeWithin(3063.954, 1); // Max upgrade Mythic+
-    expect(spoilsOfNeltharus(447)).toBeWithin(3146.328, 1); // Vault
+    expect(spoilsOfNeltharus(382)).toBeWithin(2209, 1); // Normal
+    expect(spoilsOfNeltharus(402)).toBeWithin(2514, 1); // Mythic+
+    expect(spoilsOfNeltharus(441)).toBeWithin(3108, 1); // Max upgrade Mythic+
+    expect(spoilsOfNeltharus(447)).toBeWithin(3199, 1); // Vault
```

Comment:
```suggestion
    expect(spoilsOfNeltharus(382)).toBeWithin(2209.46, 1); // Normal
    expect(spoilsOfNeltharus(402)).toBeWithin(2513.98, 1); // Mythic+
    expect(spoilsOfNeltharus(441)).toBeWithin(3107.79, 1); // Max upgrade Mythic+
    expect(spoilsOfNeltharus(447)).toBeWithin(3199.14, 1); // Vault
```

![image](https://github.com/user-attachments/assets/c530f4c4-816b-43be-a9c5-9a063cadd555)
![image](https://github.com/user-attachments/assets/047bdb04-e68d-4338-83fd-db6cfb21f4dd)
![image](https://github.com/user-attachments/assets/19dd476d-372c-459c-8822-d4745622b690)
![image](https://github.com/user-attachments/assets/0bf445c3-a3f4-403e-b516-b5cecbd7581f)